from PySide6.QtCore import (QCoreApplication, QMetaObject, QRect, Qt)
from PySide6.QtGui import (QBrush, QColor, QPalette)
from PySide6.QtWidgets import (QLabel, QLineEdit, QPushButton)


class Ui_EmployeeLogin(object):
    def setupUi(self, EmployeeLogin):
        if not EmployeeLogin.objectName():
            EmployeeLogin.setObjectName(u"EmployeeLogin")
        EmployeeLogin.resize(293, 249)
        palette = QPalette()
        brush = QBrush(QColor(255, 227, 203, 255))
        brush.setStyle(Qt.SolidPattern)
        palette.setBrush(QPalette.Active, QPalette.WindowText, brush)
        brush1 = QBrush(QColor(26, 110, 175, 255))
        brush1.setStyle(Qt.SolidPattern)
        palette.setBrush(QPalette.Active, QPalette.Button, brush1)
        brush2 = QBrush(QColor(136, 136, 136, 255))
        brush2.setStyle(Qt.SolidPattern)
        palette.setBrush(QPalette.Active, QPalette.Light, brush2)
        palette.setBrush(QPalette.Active, QPalette.Text, brush)
        palette.setBrush(QPalette.Active, QPalette.ButtonText, brush)
        brush3 = QBrush(QColor(0, 0, 0, 255))
        brush3.setStyle(Qt.SolidPattern)
        palette.setBrush(QPalette.Active, QPalette.Base, brush3)
        brush4 = QBrush(QColor(120, 131, 170, 255))
        brush4.setStyle(Qt.SolidPattern)
        palette.setBrush(QPalette.Active, QPalette.Window, brush4)
        palette.setBrush(QPalette.Active, QPalette.Shadow, brush)
        palette.setBrush(QPalette.Active, QPalette.HighlightedText, brush)
        brush5 = QBrush(QColor(255, 255, 255, 255))
        brush5.setStyle(Qt.SolidPattern)
        palette.setBrush(QPalette.Active, QPalette.ToolTipBase, brush5)
        palette.setBrush(QPalette.Active, QPalette.ToolTipText, brush5)
        brush6 = QBrush(QColor(85, 0, 255, 128))
        brush6.setStyle(Qt.SolidPattern)
#if QT_VERSION >= QT_VERSION_CHECK(5, 12, 0)
        palette.setBrush(QPalette.Active, QPalette.PlaceholderText, brush6)
#endif
        palette.setBrush(QPalette.Inactive, QPalette.WindowText, brush)
        palette.setBrush(QPalette.Inactive, QPalette.Button, brush1)
        brush7 = QBrush(QColor(255, 85, 255, 255))
        brush7.setStyle(Qt.SolidPattern)
        palette.setBrush(QPalette.Inactive, QPalette.Window, brush7)
        palette.setBrush(QPalette.Inactive, QPalette.HighlightedText, brush)
        palette.setBrush(QPalette.Inactive, QPalette.ToolTipBase, brush5)
        palette.setBrush(QPalette.Disabled, QPalette.Text, brush5)
        palette.setBrush(QPalette.Disabled, QPalette.ButtonText, brush5)
        palette.setBrush(QPalette.Disabled, QPalette.HighlightedText, brush)
        EmployeeLogin.setPalette(palette)
        self.EmployeeNameLogin = QLineEdit(EmployeeLogin)
        self.EmployeeNameLogin.setObjectName(u"EmployeeNameLogin")
        self.EmployeeNameLogin.setGeometry(QRect(90, 60, 113, 22))
        self.EmployeePassword = QLineEdit(EmployeeLogin)
        self.EmployeePassword.setObjectName(u"EmployeePassword")
        self.EmployeePassword.setGeometry(QRect(90, 130, 113, 22))
        self.EmployeePassword.setEchoMode(QLineEdit.EchoMode.Password)
        self.EmployeeNameLabel = QLabel(EmployeeLogin)
        self.EmployeeNameLabel.setObjectName(u"EmployeeNameLabel")
        self.EmployeeNameLabel.setGeometry(QRect(90, 30, 91, 16))
        self.EmployeePasswordLabel = QLabel(EmployeeLogin)
        self.EmployeePasswordLabel.setObjectName(u"EmployeePasswordLabel")
        self.EmployeePasswordLabel.setGeometry(QRect(90, 100, 51, 16))
        self.LoginButton = QPushButton(EmployeeLogin)
        self.LoginButton.setObjectName(u"LoginButton")
        self.LoginButton.setGeometry(QRect(170, 180, 75, 24))
        self.LogOutButton = QPushButton(EmployeeLogin)
        self.LogOutButton.setObjectName(u"LogOutButton")
        self.LogOutButton.setGeometry(QRect(50, 180, 75, 24))
        self.CreateEmployeeButton = QPushButton(EmployeeLogin)
        self.CreateEmployeeButton.setObjectName(u"CreateEmployeeButton")
        self.CreateEmployeeButton.setGeometry(QRect(100, 210, 101, 24))

        self.retranslateUi(EmployeeLogin)

        QMetaObject.connectSlotsByName(EmployeeLogin)
    # setupUi

    def retranslateUi(self, EmployeeLogin):
        EmployeeLogin.setWindowTitle(QCoreApplication.translate("EmployeeLogin", u"Form", None))
        self.EmployeeNameLabel.setText(QCoreApplication.translate("EmployeeLogin", u"Employee Name", None))
        self.EmployeePasswordLabel.setText(QCoreApplication.translate("EmployeeLogin", u"Password", None))
        self.LoginButton.setText(QCoreApplication.translate("EmployeeLogin", u"Log In", None))
        self.LogOutButton.setText(QCoreApplication.translate("EmployeeLogin", u"Log Out", None))
        self.CreateEmployeeButton.setText(QCoreApplication.translate("EmployeeLogin", u"Create Employee", None))
    # retranslateUi

